scripts/mytop.sh

```diff
@@ -147,7 +147,12 @@ if (-e $config)
 
             if (/(\S+)\s*=\s*(.*\S)/)
             {
-                $config{lc $1} = $2 if exists $config{lc $1};
+                my ($k, $v) = ($1, $2);
+                if ($k =~ /^filter_/i) {
+                  $config{lc $k} = StringOrRegex($v) if exists $config{lc $k};
```

Comment:
Perhaps worth adding a comment that references where these configuration keys are handled in the mainloop, and the fact that they need to store regex objects rather than string literals in order to work correctly?

https://github.com/MariaDB/server/blob/43f1f9343efad7d29c32aa9c5649b8b0a2c221d7/scripts/mytop.sh#L529-L573